docs/walkthrough.md

```diff
@@ -217,7 +217,7 @@ Deployment has permission to list resources in the cluster:
 
 ```shell
 $ kubectl create clusterrole resource-lister --verb=list --resource="*"
```

Comment:
Actually `--resource="*"` does not work on v1.10.0. I sent a patch https://github.com/kubernetes/kubernetes/pull/62945, though.